app/src/main/java/com/monkeyteam/chimpagne/model/database/ChimpagneAccountManager.kt

```diff
@@ -46,124 +43,193 @@ class ChimpagneAccountManager(
   }
 
   fun getAccountWithProfilePicture(
-    uid: ChimpagneAccountUID,
-    onSuccess: (ChimpagneAccount?, Uri?) -> Unit,
-    onFailure: (Exception) -> Unit
+      uid: ChimpagneAccountUID,
+      onSuccess: (ChimpagneAccount?, Uri?) -> Unit,
+      onFailure: (Exception) -> Unit
   ) {
-    getAccount(uid, { account ->
-      if (account == null) onSuccess(null, null)
-      else downloadProfilePicture(
-        account.firebaseAuthUID,
-      ) { uri -> onSuccess(account, uri) }
-    }, onFailure)
+    getAccount(
+        uid,
+        { account ->
+          if (account == null) onSuccess(null, null)
+          else
+              downloadProfilePicture(
+                  account.firebaseAuthUID,
+              ) { uri ->
+                onSuccess(account, uri)
+              }
+        },
+        onFailure)
   }
 
   /** Puts the given account to Firebase and updates [currentUserAccount] accordingly */
   fun updateCurrentAccount(
-    account: ChimpagneAccount, onSuccess: () -> Unit, onFailure: (Exception) -> Unit
+      account: ChimpagneAccount,
+      onSuccess: () -> Unit,
+      onFailure: (Exception) -> Unit
   ) {
-    accounts.document(account.firebaseAuthUID).set(account).addOnSuccessListener {
-      currentUserAccount = account
-      onSuccess()
-    }.addOnFailureListener { onFailure(it) }
+    accounts
+        .document(account.firebaseAuthUID)
+        .set(account)
+        .addOnSuccessListener {
+          currentUserAccount = account
+          onSuccess()
+        }
+        .addOnFailureListener { onFailure(it) }
   }
 
   fun updateCurrentAccount(
-    account: ChimpagneAccount,
-    profilePicture: Uri?,
-    onSuccess: () -> Unit,
-    onFailure: (Exception) -> Unit
+      account: ChimpagneAccount,
+      profilePicture: Uri?,
+      onSuccess: () -> Unit,
+      onFailure: (Exception) -> Unit
   ) {
     if (profilePicture == null) {
       updateCurrentAccount(account, onSuccess, onFailure)
     } else {
-      uploadProfilePicture(account, profilePicture, {
-        updateCurrentAccount(account, onSuccess, onFailure)
-      }, onFailure)
+      uploadProfilePicture(
+          account,
+          profilePicture,
+          { updateCurrentAccount(account, onSuccess, onFailure) },
+          onFailure)
     }
   }
 
   private fun uploadProfilePicture(
-    account: ChimpagneAccount, uri: Uri, onSuccess: (String) -> Unit, onFailure: (Exception) -> Unit
+      account: ChimpagneAccount,
+      uri: Uri,
+      onSuccess: (String) -> Unit,
+      onFailure: (Exception) -> Unit
   ) {
-//    profilePictures.child(account.firebaseAuthUID).delete()
+    //    profilePictures.child(account.firebaseAuthUID).delete()
     val imageRef = profilePictures.child(account.firebaseAuthUID)
-    imageRef.putFile(uri).addOnSuccessListener {
-      imageRef.downloadUrl.addOnSuccessListener { downloadUrl ->
-        Log.d("ChimpagneAccountManager", "Uploaded image to: $downloadUrl")
```

Comment:
I don't think logs used when developing should be in the main branch

Reply:
not part of this PR

Reply:
ChimpagnesAccountManager.kt should be pulled from main before merging then.